src/dymoprint/lib/unicode_blocks.py

```diff
@@ -5,25 +5,35 @@
 LH = "▄"
 FB = "█"
 NB = "\xa0"
+LS = "░"
 
 assert UH == "\N{UPPER HALF BLOCK}"
 assert LH == "\N{LOWER HALF BLOCK}"
 assert FB == "\N{FULL BLOCK}"
 assert NB == "\N{NO-BREAK SPACE}"
+assert LS == "\N{LIGHT SHADE}"
```

Comment:
Unfortunately it doesn't seem like there are unicode block characters that shade the upper and lower half independently.

It seems to me like we're probably better off leaving the transparent pixels unshaded.

Unless I'm confused!

```suggestion
```

Reply:
I just added the shade for debugging purposes (to see where it is used), and probably forgot to remove it.